### PR TITLE
RELEASE: Update authors for v1.14 release

### PR DESCRIPTION
**Pull request by brminich**
## What
Update `AUTHORS` file for v1.14 release

### CODE REVIEW

**Comment by brminich** [comment]
@yosefe

File: AUTHORS [hunk 1]
```diff
@@ -60,23 +68,28 @@ Peter Andreas Entschev <peter@entschev.com>
...
+Robert Dietrich <rdietrich@nvidia.com>
 Rohit Zambre <rzambre@uci.edu>
+Sam James <sam@gentoo.org>
 Sasha Kotchubievsky <sashakot@mellanox.com>
 Scott Saulters <ssaulters@nvidia.com>
 Sergey Lebedev <sergeyle@mellanox.com>
 Sergey Oblomov <sergeyo@mellanox.com>
 Sergey Shalnov <sergeysh@mellanox.com>
 Serguei Sagalovitch <Serguei.Sagalovitch@amd.com>
+Shachar Hasson <shasson5@gmail.com>
```
Comment (yosefe) [inline AUTHORS:80]: pls use nvidia email

Comment (brminich) [reply AUTHORS:80]: @shasson5 uses `gmail` in git config

Comment (yosefe) [reply AUTHORS:80]: nevertheless, pls change it to nvidia
@shasson5 pls fix your git config

Comment (brminich) [reply AUTHORS]: @yosefe fixed in place

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@brminich we need to port this to master branch as well..

**Pull request merged**